RubberduckTests/Inspections/ExcessiveInterfaceMembersInspectionTests.cs

```diff
@@ -0,0 +1,127 @@
+﻿using System.Linq;
+using NUnit.Framework;
+using Moq;
+using Rubberduck.SettingsProvider;
+using Rubberduck.VBEditor.SafeComWrappers;
+using RubberduckTests.Mocks;
+using Rubberduck.Parsing.VBA;
+using System.Collections.Generic;
+using Rubberduck.CodeAnalysis.Inspections;
+using Rubberduck.CodeAnalysis.Inspections.Concrete;
+
+namespace RubberduckTests.Inspections 
+{
+    class ExcessiveInterfaceMembersInspectionTests : InspectionTestsBase 
+    {
+        private static List<(string, string, ComponentType)> modules = new List<(string, string, ComponentType)> {
```

Comment:
Static mutable state does not belong in tests. That couples the tests and makes running them in parallel impossible. I.e. running them in parallel will have undefined behaviour.